<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listToBe">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>191</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listCrumFin">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>191</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listLRFin">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>191</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listCry">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>191</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelToBe">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>To Be Finished</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCrumFin">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Finishing (Crum)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLRFin">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Finishing (L.R.)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCryFIn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Finishing (Cryst)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuGroup">
    <property name="title">
     <string>Group</string>
    </property>
    <addaction name="actionGroup_Selected"/>
   </widget>
   <widget class="QMenu" name="menuFinish">
    <property name="title">
     <string>Finish</string>
    </property>
    <widget class="QMenu" name="menuFinish_Selected">
     <property name="title">
      <string>Finish Selected</string>
     </property>
     <addaction name="actionCrumble_Goo"/>
     <addaction name="actionLive_Resin"/>
     <addaction name="actionCrystals"/>
    </widget>
    <addaction name="menuFinish_Selected"/>
   </widget>
   <widget class="QMenu" name="menuClose">
    <property name="title">
     <string>Close</string>
    </property>
    <widget class="QMenu" name="menuClose_Selected">
     <property name="title">
      <string>Close Selected</string>
     </property>
     <addaction name="actionCrum_Goo"/>
     <addaction name="actionLive_Resin_2"/>
     <addaction name="actionCrystals_2"/>
    </widget>
    <addaction name="menuClose_Selected"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGroup"/>
   <addaction name="menuFinish"/>
   <addaction name="menuClose"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionGroup_Selected">
   <property name="text">
    <string>Group Selected</string>
   </property>
  </action>
  <action name="actionCrumble_Goo">
   <property name="text">
    <string>Crumble/Goo</string>
   </property>
  </action>
  <action name="actionLive_Resin">
   <property name="text">
    <string>Live Resin</string>
   </property>
  </action>
  <action name="actionCrystals">
   <property name="text">
    <string>Crystals</string>
   </property>
  </action>
  <action name="actionTo_Be_Finished">
   <property name="text">
    <string>To Be Finished</string>
   </property>
  </action>
  <action name="actionCrumble_Finsihing">
   <property name="text">
    <string>Crumble Finsihing</string>
   </property>
  </action>
  <action name="actionLive_Resin_Finishing">
   <property name="text">
    <string>Live Resin Finishing</string>
   </property>
  </action>
  <action name="actionCrystal_Finishing">
   <property name="text">
    <string>Crystal Finishing</string>
   </property>
  </action>
  <action name="actionView_Selected">
   <property name="text">
    <string>View Selected</string>
   </property>
  </action>
  <action name="actionHarvest">
   <property name="text">
    <string>Harvest</string>
   </property>
  </action>
  <action name="actionCrum_Goo">
   <property name="text">
    <string>Crum/Goo</string>
   </property>
  </action>
  <action name="actionLive_Resin_2">
   <property name="text">
    <string>Live Resin</string>
   </property>
  </action>
  <action name="actionCrystals_2">
   <property name="text">
    <string>Crystals</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
